packages/esm-outpatient-app/src/current-visit/visit-details/vitals.component.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { Button, Grid, Row, Tag, Tile } from 'carbon-components-react';
+import { useTranslation } from 'react-i18next';
+import { PatientVitals, calculateBMI } from '../current-visit.resource';
+import ArrowRight16 from '@carbon/icons-react/es/arrow--right/16';
+import { navigate } from '@openmrs/esm-framework';
+import styles from './triage-note.scss';
+import CircleFillGlyph from '@carbon/icons-react/es/circle--solid/16';
+interface VitalsComponentProps {
+  vitals: Array<PatientVitals>;
+  patientUuid: string;
+}
+
+const Vitals: React.FC<VitalsComponentProps> = ({ vitals, patientUuid }) => {
+  const { t } = useTranslation();
+
+  let formattedVitals = vitals.reduce((r, c) => Object.assign(r, c), {});
+
+  return (
+    <div>
+      {formattedVitals ? (
+        <div>
+          <Grid className={styles.grid}>
+            <Row>
+              <Tile light>
+                <p>Temperature</p>
+                <div className={styles.vitalValuesWrapper}>
+                  <p className={styles.vitalValues}>
+                    {formattedVitals.temperature ? formattedVitals.temperature : '--'}
+                  </p>
+                  <p className={styles.unit}>°C</p>
+                </div>
+              </Tile>
+              <Tile light>
+                <p>Bp</p>
+                <div className={styles.vitalValuesWrapper}>
+                  <p className={styles.vitalValues}>{formattedVitals.systolic ? formattedVitals.systolic : '--'}</p>
+                  <p className={styles.unit}> mmHg</p>
```

Comment:
Translation

Reply:
We should extract this  unit symbol from the concept, this is to ensure that when you have saved weight in either pounds or kilograms the correct symbol is always displayed